### PR TITLE
Add package bxa

### PR DESCRIPTION
**Pull request by JohannesBuchner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

python `@conda-forge/help-python`

### CODE REVIEW

**Comment by JohannesBuchner** [comment]
@xhochy I don't know what you did in https://github.com/conda-forge/staged-recipes/pull/10753 to fix the SSL verification issue, but could you do that here too?

File: recipes/bxa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  # may need to later add "skip: True  # [win]" to disable Windows.
+  skip: true  # [win or py<30]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - numpy
```
Comment (xhochy) [inline recipes/bxa/meta.yaml]: Is this really required? As you are not compiling code, `numpy`  can probably be omitted.

File: recipes/bxa/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - ultranest
+    - pymultinest
+    - multinest
+
+test:
+  imports:
```
Comment (xhochy) [inline recipes/bxa/meta.yaml]: Only test the package itself, omit `pymultinest` and `ultranest`.

**Comment by xhochy** [comment]
`noarch: python` so Windows can be ignored.

**Pull request merged**